### ISSUE TITLE
Installer fails

### ISSUE THREAD

**Issue by beppe9000** [jgm/pandoc#2845]
The installer fails on windows 10 when trying to install for "all users" with this silly error. 
*AFAIK the` C:\ `drive is local enough to me.*
![cyberfox_2016-04-07_01-11-05](https://cloud.githubusercontent.com/assets/6625123/14335716/c44563ba-fc5d-11e5-86e7-990fb7d06483.png)

**Comment by jgm** [comment]
@nkalvi any ideas what is going on here?

**Comment by nkalvi** [comment]
@beppe9000 Could you try the tip here: http://ephams.com/2013/03/10/msi-installer-error-installation-directory-local-hard-drive/

i.e. From a command prompt as admin enter
`MSIexec /i [path to the Pandoc installation program]`

@jgm It doesn't look like the issue is particular to Pandoc's MSI; I'll await to see whether the alternate installation method worked.

**Comment by beppe9000** [comment]
I did not mention (in the first post) that I was able to complete the setup selecting the "install only for me" option... this directly points at a permissions issue.

@jgm Meanwhile I investigated the problem as other setup program failed and I discovered (thanks to MATLAB) that the %TEMP%  folder had the permssions changed by some other program (many people on the internet blame acrobat 11, but i never used it).

As a workaround I started running everything as administrator but due to the security concerns it was only a temporary workaround. I finally solved it by using takeown and setting the permissions of the %TEMP% folder giving "complete control" for the "Everyone" user.

I close this issue as it was not pandoc's fault, but I suggest for you to include a check on the writability of the %TEMP% folder and elevate the installation if it is not, as a measure against improper configuration (wanted or unwanted) on the user end.
